### DISCUSSION TITLE
Step by step prediction?

### DISCUSSION THREAD

**Discussion by monk1337** [pyg-team/pytorch_geometric#6796] [General]
I have a problem statement. I take a few inputs from the user, for example, I take "feature_a" and "feature_b" as input from the user. Now based on "feature_a", and "feature_b" I show options for "feature_3" and let the user select a few options. After selecting a few options from "feature_3" I suggest options for "feature_4" based on "featur_1", "feature_2", and "feature_3" variables. After the user selects a few options from "feature_4" Now my data looks like this

"feature_1", "featue_2", "feature_3", "feature_4" 

Now based on all these features I predict a class.

Can I solve this problem with Pytorch Geometric?

**Comment by rusty1s** [comment]
I think this is doable. Whether PyG is the right tool for the given job depends though on whether there exists a graph. If you just have single flattened features of a user, there is no need to use PyG.
